sdk/storage/Azure.Storage.Files.DataLake/CHANGELOG.md

```diff
@@ -1,14 +1,8 @@
 # Release History
 
 ## 12.9.0-beta.2 (Unreleased)
-
-### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
-
-### Other Changes
+- Added supprot for service version 2021-02-12.
+- Added support for listing system file systems with DataLakeServiceClient.GetFileSystmes() and .GetFileSystemsAsync().
```

Comment:
DataLakeServiceClient.GetFileSystmes() should be DataLakeServiceClient.GetFileSystems()

Reply:
Fixed.